2018/_data/talks.json

```diff
@@ -129,6 +131,8 @@
         "best practices"
       ],
       "level": "beginner",
+      "file": "JBCN_DIA 12/SALA 13/180612_JBCN_RUSTAM MEHMANDAROV_13_1.mp4",
```

Comment:
What's the purpose of the "file" property ? I haven't seen anywhere its usage

Reply:
It was used the upload videos script, I used this field for that, otherwise the videos could not be uploaded ;-)